pkg/skaffold/render/renderer/renderer_test.go

```diff
@@ -132,6 +133,7 @@ pipeline:
 
 			var b bytes.Buffer
 			err = r.Render(context.Background(), &b, []graph.Artifact{{ImageName: "leeroy-web", Tag: "leeroy-web:v1"}})
+			defer os.RemoveAll(".kpt-pipeline")
```

Comment:
Is this file going to be generated in the current dir? Instead can we generate this file in the root of project? 
In this case its it the temp dir

Reply:
This is not needed. Didn't realize the NewTempDir has a deferred cleanup